api_tests/Gemfile.lock

```diff
@@ -36,5 +36,8 @@ DEPENDENCIES
   pry
   rspec
 
+RUBY VERSION
```

Comment:
did you change this yourself, or auto updated?

Reply:
@GuyHarwood this is added automatically as now i have added a ruby version to the Gemfile. Then when we run bundle install, it fails if any other version of ruby is used with the following error message:
"Your Ruby version is 2.4.0, but your Gemfile specified 2.6.0"

Reply:
Good 👍